File: data structures and algorithms/python/06_054.py
def test_performance():
    large_arr = [i % 100 for i in range(1000000)]
    k = 1000
    start_time = time.time()
    result = maxSumSubarray(large_arr, k)
    end_time = time.time()
    print(f"Time taken: {end_time - start_time} seconds")

File: data structures and algorithms/cs/06_054.cs
using System;

public static class TestPerformance
{
   public static void TestPerformanceMethod()
   {
      int[] largeArr = new int[1000000];
      for (int i = 0; i < largeArr.Length; i++)
      {
         largeArr[i] = i % 100;
      }
      int k = 1000;
      long startTime = DateTimeOffset.Now.ToUnixTimeMilliseconds();
      int result = MaxSumSubarray(largeArr, k);
      long endTime = DateTimeOffset.Now.ToUnixTimeMilliseconds();
      Console.WriteLine("Time taken: " +
         (endTime - startTime) / 1000.0 + " seconds");
   }
}